### PR TITLE
Install FFMPEG version 6

### PR DESCRIPTION
**Pull request by mjh1**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -74,6 +75,7 @@ RUN	apt update && apt install -yqq \
 ADD ./scripts/livepeer-vmagent /usr/local/bin
 COPY --from=catalyst-build	/opt/bin/		/usr/local/bin/
 COPY --from=node-build		/app/go-tools/w3	/opt/local/lib/livepeer-w3
+COPY --from=ffmpeg-build	/usr/local/bin/*		/usr/local/bin/
```
Comment (mjh1) [inline Dockerfile:78]: @hjpotter92 I can't seem to get this to work, I've added the copy in multiple places but still not seeing the binaries when i run livepeer in a box locally. any idea how i should be doing this?

Comment (iameli-streams) [reply Dockerfile:78]: `/usr/local/bin` gets overridden by `make box-dev` and mounts your local `bin` directory instead - that's how binaries built outside the container get in. You could try this?
```suggestion
COPY --from=ffmpeg-build	/usr/local/bin/*		/usr/bin/
```
Buuuuuut that still won't work with inplace-deploy. We're either going to need to ship a static ffmpeg alongside the other binaries or statically link it like we're doing with go-livepeer.

File: Dockerfile [hunk 2]
```diff
@@ -49,6 +49,8 @@ RUN	npm install --prefix /app/go-tools/w3
 # chown needed to make everything owned by one user for userspace podman execution
 RUN	chown -R root:root /app/go-tools/w3
 
+FROM linuxserver/ffmpeg:version-6.0-cli as ffmpeg-build
```
Comment (iameli-streams) [inline Dockerfile:52]: This ffmpeg build is extremely dynamically linked:
```
# ldd /usr/local/bin/ffmpeg
	linux-vdso.so.1 (0x00007ffc67544000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f96130df000)
	libxcb.so.1 => /lib/x86_64-linux-gnu/libxcb.so.1 (0x00007f96130b5000)
	libxcb-shape.so.0 => /lib/x86_64-linux-gnu/libxcb-shape.so.0 (0x00007f96130b0000)
	libxcb-xfixes.so.0 => /lib/x86_64-linux-gnu/libxcb-xfixes.so.0 (0x00007f96130a6000)
	libv4l2.so.0 => /lib/x86_64-linux-gnu/libv4l2.so.0 (0x00007f9613096000)
	libvmaf.so.1 => /usr/local/lib/libvmaf.so.1 (0x00007f9612f81000)
	libass.so.9 => /usr/local/lib/libass.so.9 (0x00007f9612f48000)
	libva.so.2 => /usr/local/lib/libva.so.2 (0x00007f9612f16000)
	libvidstab.so.1.2 => /usr/local/lib/libvidstab.so.1.2 (0x00007f9612efe000)
	libOpenCL.so.1 => /lib/x86_64-linux-gnu/libOpenCL.so.1 (0x00007f9612eec000)
	libfreetype.so.6 => /usr/local/lib/libfreetype.so.6 (0x00007f9612e2f000)
	libvpl.so.2 => /usr/local/lib/libvpl.so.2 (0x00007f9612deb000)
	libxml2.so.2 => /lib/x86_64-linux-gnu/libxml2.so.2 (0x00007f9612c09000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f9612bed000)
	libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f9612b49000)
	libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f9612707000)
	libvpx.so.8 => /usr/local/lib/libvpx.so.8 (0x00007f9612403000)
	libwebpmux.so.3 => /usr/local/lib/libwebpmux.so.3 (0x00007f96123f4000)
	libopencore-amrwb.so.0 => /usr/local/lib/libopencore-amrwb.so.0 (0x00007f96123de000)
	libfdk-aac.so.2 => /usr/local/lib/libfdk-aac.so.2 (0x00007f9612296000)
	libmp3lame.so.0 => /usr/local/lib/libmp3lame.so.0 (0x00007f96121fe000)
	libopencore-amrnb.so.0 => /usr/local/lib/libopencore-amrnb.so.0 (0x00007f96121d3000)
	libopenjp2.so.7 => /usr/local/lib/libopenjp2.so.7 (0x00007f9612143000)
	libopus.so.0 => /usr/local/lib/libopus.so.0 (0x00007f96120e3000)
	libSvtAv1Enc.so.1 => /usr/local/lib/libSvtAv1Enc.so.1 (0x00007f961197e000)
	libtheoraenc.so.1 => /usr/local/lib/libtheoraenc.so.1 (0x00007f961193e000)
	libtheoradec.so.1 => /usr/local/lib/libtheoradec.so.1 (0x00007f961191e000)
	libvorbis.so.0 => /usr/local/lib/libvorbis.so.0 (0x00007f96118e2000)
	libvorbisenc.so.2 => /usr/local/lib/libvorbisenc.so.2 (0x00007f9611836000)
	libwebp.so.7 => /usr/local/lib/libwebp.so.7 (0x00007f96117c5000)
	libx264.so.164 => /usr/local/lib/libx264.so.164 (0x00007f96114a5000)
	libx265.so.199 => /usr/local/lib/libx265.so.199 (0x00007f96101de000)
	libxvidcore.so.4 => /usr/local/lib/libxvidcore.so.4 (0x00007f96100d0000)
	libkvazaar.so.7 => /usr/local/lib/libkvazaar.so.7 (0x00007f9610033000)
	libva-drm.so.2 => /usr/local/lib/libva-drm.so.2 (0x00007f961002e000)
	libva-x11.so.2 => /usr/local/lib/libva-x11.so.2 (0x00007f9610024000)
	libvdpau.so.1 => /usr/local/lib/libvdpau.so.1 (0x00007f961001e000)
	libX11.so.6 => /lib/x86_64-linux-gnu/libX11.so.6 (0x00007f960fede000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f960fcb6000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f96150ad000)
	libXau.so.6 => /lib/x86_64-linux-gnu/libXau.so.6 (0x00007f960fcb0000)
	libXdmcp.so.6 => /lib/x86_64-linux-gnu/libXdmcp.so.6 (0x00007f960fca8000)
	libv4lconvert.so.0 => /lib/x86_64-linux-gnu/libv4lconvert.so.0 (0x00007f960fc2b000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f960f9ff000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f960f9df000)
	libfribidi.so.0 => /usr/local/lib/libfribidi.so.0 (0x00007f960f9bf000)
	libharfbuzz.so.0 => /lib/x86_64-linux-gnu/libharfbuzz.so.0 (0x00007f960f8f0000)
	libfontconfig.so.1 => /usr/local/lib/libfontconfig.so.1 (0x00007f960f8a4000)
	libgomp.so.1 => /lib/x86_64-linux-gnu/libgomp.so.1 (0x00007f960f858000)
	libpng16.so.16 => /usr/local/lib/libpng16.so.16 (0x00007f960f810000)
	libbrotlidec.so.1 => /lib/x86_64-linux-gnu/libbrotlidec.so.1 (0x00007f960f802000)
	libicuuc.so.70 => /lib/x86_64-linux-gnu/libicuuc.so.70 (0x00007f960f607000)
	liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5 (0x00007f960f5dc000)
	libogg.so.0 => /usr/local/lib/libogg.so.0 (0x00007f960f5cf000)
	libmvec.so.1 => /lib/x86_64-linux-gnu/libmvec.so.1 (0x00007f960f4d2000)
	libsharpyuv.so.0 => /usr/local/lib/libsharpyuv.so.0 (0x00007f960f4ca000)
	libdrm.so.2 => /usr/local/lib/x86_64-linux-gnu/libdrm.so.2 (0x00007f960f4b2000)
	libXext.so.6 => /lib/x86_64-linux-gnu/libXext.so.6 (0x00007f960f49d000)
	libXfixes.so.3 => /lib/x86_64-linux-gnu/libXfixes.so.3 (0x00007f960f493000)
	libX11-xcb.so.1 => /lib/x86_64-linux-gnu/libX11-xcb.so.1 (0x00007f960f48e000)
	libxcb-dri3.so.0 => /lib/x86_64-linux-gnu/libxcb-dri3.so.0 (0x00007f960f487000)
	libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f960f46f000)
	libjpeg.so.8 => /lib/x86_64-linux-gnu/libjpeg.so.8 (0x00007f960f3ee000)
	libglib-2.0.so.0 => /lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f960f2b2000)
	libgraphite2.so.3 => /lib/x86_64-linux-gnu/libgraphite2.so.3 (0x00007f960f28b000)
	libexpat.so.1 => /lib/x86_64-linux-gnu/libexpat.so.1 (0x00007f960f25a000)
	libbrotlicommon.so.1 => /lib/x86_64-linux-gnu/libbrotlicommon.so.1 (0x00007f960f237000)
	libicudata.so.70 => /lib/x86_64-linux-gnu/libicudata.so.70 (0x00007f960d619000)
	libmd.so.0 => /lib/x86_64-linux-gnu/libmd.so.0 (0x00007f960d60a000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f960d594000)
```

It won't work without alllll of those files; we need a static one

**Comment by mjh1** [comment]
Thanks Eli, closing this in favour of https://github.com/livepeer/catalyst/pull/711

**Pull request closed without merging**